roles/tarzan/tasks/dependencies.yml

```diff
@@ -114,6 +52,46 @@
     chdir: "{{ openresty_dest}}/openresty-{{ openresty_version }}" 
     creates: "{{ openresty_dest }}/{{ openresty_version }}/bin/resty"
 
+- name: create luarocks destination directory
+  file: path={{ luarocks_dest }} state=directory mode=g+rwXs
+
+- name: download luarocks archive
+  get_url:
+    url: "{{ luarocks_url }}"
+    dest: "{{ luarocks_dest }}/{{ luarocks_file }}"
+
+- name: unpack luarocks to destination
+  unarchive: 
+    src: "{{ luarocks_dest }}/{{ luarocks_file }}" 
+    dest: "{{ luarocks_dest }}"
+    copy: no 
+    creates: "{{ luarocks_dest }}/{{ luarocks_version }}/bin/luarocks"
+    mode: g+rwX,o+rX
+
+- name: figure out luajit version/include folder name
+  find:
+    paths: "{{ openresty_dest }}/{{ openresty_version }}/luajit/include"
+    patterns: "luajit-*"
+    recurse: no
+    file_type: directory
+  register: luajit_include_folder
+
+- name: make sure the include folder is unambiguous
+  assert:
+    that: "luajit_include_folder.matched == 1"
```

Comment:
Okay, like I feared, the shell-globbing didn't work. I changed it so that I use ansible's `find` module to get the luajit include folder name and a basic sanity check that makes sure it is only one folder

Reply:
Well, I think its more better this way :)